### PR TITLE
#165087316 Modularised codebase

### PR DESCRIPTION
**Pull request by oldmonad**
#### Screenshots

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
   "scripts": {
     "start": "node build/server.js",
     "start:dev": "nodemon --exec babel-node src/server.js",
-    "test": "NODE_ENV=test nyc mocha src/tests/**/**.test.js --require @babel/register --exit --no-timeouts",
+    "test": "NODE_ENV=test nyc mocha --recursive src/tests --require @babel/register --exit --no-timeouts",
     "test:watch": "NODE_ENV=test mocha src/tests/**/**.test.js --watch",
```
Comment (eben-k) [inline package.json:21]: Could we have the '--recursive' here as well?

File: src/routes/utils/jwt.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    if (typeof header === 'undefined') {
+      return errorResponse(res, 401, 'You are not authorized to make this action');
+    }
+
+    const token = header.split(' ')[1];
+    if (!token) {
+      return errorResponse(res, 401, 'You are not authorized to make this action please login');
+    }
+    try {
+      const decoded = await jwt.verify(token, process.env.SECRET_KEY);
```
Comment (musonant) [inline src/routes/utils/jwt.js]: This line seems to be doing the same thing as the `verifyToken` method just above.

**Review by Japhethca — APPROVED** [review]
Looks good! 
Though there were no tests for the functions.

**Comment by oldmonad** [comment]
Ok I would add tests for those functions

**Pull request merged**